src/main/java/ch/uzh/ifi/seal/soprafs20/service/LobbyService.java

```diff
@@ -0,0 +1,127 @@
+package ch.uzh.ifi.seal.soprafs20.service;
+
+
+import ch.uzh.ifi.seal.soprafs20.entity.Lobby;
+import ch.uzh.ifi.seal.soprafs20.entity.User;
+import ch.uzh.ifi.seal.soprafs20.exceptions.LobbyException;
+import ch.uzh.ifi.seal.soprafs20.repository.LobbyRepository;
+import ch.uzh.ifi.seal.soprafs20.repository.UserRepository;
+import ch.uzh.ifi.seal.soprafs20.rest.dto.LobbyGetDTO;
+import ch.uzh.ifi.seal.soprafs20.rest.mapper.DTOMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * Lobby Service
+ * This class is the "worker" and responsible for all functionality related to the lobby
+ * (e.g., it creates, modifies). The result will be passed back to the caller.
+ */
+
+@Service
+@Transactional
+public class LobbyService {
+
+
+    private final Logger log = LoggerFactory.getLogger(LobbyService.class);
+
+    private final LobbyRepository lobbyRepository;
+    private final UserRepository userRepository;
+
+    @Autowired
+    public LobbyService(@Qualifier("lobbyRepository") LobbyRepository lobbyRepository, @Qualifier("userRepository") UserRepository userRepository) {
+        this.lobbyRepository = lobbyRepository;
+        this.userRepository = userRepository;
+    }
+
+    public Lobby createLobby(Lobby newLobby){
+
+        checkIfLobbyExist(newLobby);
+        lobbyRepository.save(newLobby);
+
+        return newLobby;
+
+    }
+
+    public void saveOrUpdate(Lobby updateLobby){
```

Comment:
Why are you making a separate function for this one line? Did you intend to `flush()` to the database as well?